tasks/vertica.yaml

```diff
@@ -36,5 +36,7 @@
     - mon_users.sql
   when: sql_file_status|changed or sql_template_status|changed
 
+- name: Create cron job to clean up old partitions
+  template: src=vertica-cleanup.j2 dest=/etc/cron.daily/vertica-cleanup mode=755
```

Comment:
These permissions are too open for a file that contains the vertica dbadmin password. I would make it 700 or 750.